spec/manager_spec.rb

```diff
@@ -0,0 +1,289 @@
+# frozen_string_literal: true
+
+require './lib/manager.rb'
+require './lib/player.rb'
+
+describe GameManager do
```

Comment:
```ruby
let(:game) { GameManager.new([Player.new('Fernando'), Player.new('Ahmed')]) }
```
By putting that instructions here, it can be use by any test written below.